plugin

```diff
@@ -158,6 +158,7 @@
   "harmonie-durrant/hha-cards",
   "hasl-sensor/lovelace-hasl-departure-card",
   "hasl-sensor/lovelace-hasl-traffic-status-card",
+  "HeWeDe/custom-gauge-card",
```

Comment:
Your release is not in english, so this is only relevant for users in a specific country, the released `hacs.json` file needs a country key.
<https://hacs.xyz/docs/publish/include>